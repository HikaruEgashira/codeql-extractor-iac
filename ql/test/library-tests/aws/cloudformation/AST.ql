private import iac

query predicate cloudformation(CloudFormation::Document n) { any() }

query predicate resources(CloudFormation::Resource n) { any() }

query predicate resourceProperties(CloudFormation::ResourceProperties n) { any() }
