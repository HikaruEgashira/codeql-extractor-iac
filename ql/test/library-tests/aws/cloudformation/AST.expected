cloudformation
| s3.json:1:1:77:1 | CloudFormation Document |
| s3.yml:1:1:42:59 | CloudFormation Document |
resources
| s3.json:4:21:15:9 | CloudFormation S3 Bucket |
| s3.json:16:25:47:9 | CloudFormation S3 Bucket Policy |
| s3.yml:4:5:12:2 | CloudFormation S3 Bucket |
| s3.yml:13:5:28:28 | CloudFormation S3 Bucket Policy |
resourceProperties
| s3.json:6:27:12:13 | CloudFormation Resource Properties |
| s3.json:18:27:46:13 | CloudFormation Resource Properties |
| s3.yml:6:7:10:4 | CloudFormation Resource Properties |
| s3.yml:15:7:28:28 | CloudFormation Resource Properties |
s3
| s3.json:4:21:15:9 | CloudFormation S3 Bucket |
| s3.yml:4:5:12:2 | CloudFormation S3 Bucket |
s3Policy
| s3.yml:4:5:12:2 | CloudFormation S3 Bucket | s3.yml:13:5:28:28 | CloudFormation S3 Bucket Policy |
